ultrasonic/lint-baseline.xml

```diff
@@ -103,6 +110,39 @@
             column="9"/>
     </issue>
 
+    <issue
+        id="ObsoleteSdkInt"
+        message="Unnecessary; SDK_INT is always >= 21"
+        errorLine1="    tools:targetApi=&quot;lollipop&quot;>"
+        errorLine2="    ~~~~~~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="src/main/res/drawable/default_ripple.xml"
+            line="6"
+            column="5"/>
+    </issue>
```

Comment:
These don't need to be hidden, they can be fixed ;)

Reply:
True. Done. :-)